multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -179,7 +179,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Manga Drop Out", "https://www.mangadropout.xyz", "id", isNsfw = true, overrideVersionCode = 1),
         SingleLang("Manga Fenix", "https://manga-fenix.com", "es", overrideVersionCode = 2),
         SingleLang("Manga Funny", "https://mangafunny.com", "en"),
-        SingleLang("Manga Gohan", "https://mangagohan.com", "ja"),
+        SingleLang("Manga Gohan", "https://gokumanga.com/", "ja", overrideVersionCode = 2),
```

Comment:
```suggestion
        SingleLang("Manga Gohan", "https://gokumanga.com/", "ja", overrideVersionCode = 1),
```

Should be `1` because the first value if not provided is `0`.

Reply:
done, and force pushed.